deploy/helm/nfd-operator/templates/deployment.yaml

```diff
@@ -7,75 +7,52 @@ metadata:
   name: {{ include "node-feature-discovery-operator.fullname" . }}-controller-manager
   namespace: {{ include "node-feature-discovery-operator.namespace" . }}
 spec:
-  replicas: 1
   selector:
     matchLabels:
       control-plane: controller-manager
       {{- include "node-feature-discovery-operator.selectorLabels" . | nindent 6 }}
+  replicas: 1
   template:
     metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
       labels:
-        {{- include "node-feature-discovery-operator.selectorLabels" . | nindent 8 }}
```

Comment:
You must not delete the selectorLabels from here as you have them in the selector above